schemas/context/profile-other-work-details.schema.json

```diff
@@ -0,0 +1,73 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/profile-other-work-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Profile Other Work Details",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
+  "description": "This mixin is used to capture information regarding additional/other work-related details of the individual such as work address, work email, work phone number, and organizations to which the profile belongs.",
+  "definitions": {
+    "profile-other-work-details": {
+      "properties": {
+        "xdm:otherAddress": {
+          "title": "Other Address",
+          "$ref": "https://ns.adobe.com/xdm/common/address",
+          "description": "Other postal address."
+        },
+        "xdm:otherPhone": {
+          "title": "Other Phone",
+          "$ref": "https://ns.adobe.com/xdm/context/phonenumber",
+          "description": "Other phone number."
+        },
+   		"xdm:assistantDetails": {
+			"xdm:name": {
+			  "title": "Assistant Name",
+			  "$ref": "https://ns.adobe.com/xdm/context/person-name",
+			  "description": "Assistant name."
+			},
+			"xdm:phone": {
+			  "title": "Assistant Phone",
+			  "$ref": "https://ns.adobe.com/xdm/context/phonenumber",
+			  "description": "Assistant phone."
+			}
+		
+        },
+        "xdm:reportsToID": {
+          "title": "Reports To Id",
+          "description": "Reports to identifier.",
+		  "type": "string"
+        },
+        "xdm:departments": {
```

Comment:
Move-In profile-work-details

Reply:
This was in the profile-work-details before. I moved these to the new mixin after we discussed.